README.md

```diff
@@ -93,46 +95,66 @@ Migrations
 ### Migrations to 0.80
 
 0.80 introduces a completely different way to manage tags for events.
+
+It is very important that you test this migration in a pre-production environment as once you drop the materialized view
+and tag columns you can not roll back.
+ 
+Most of the migration can be done while the old version of your application is running. The remaining steps happen automatically
+when a persistent actor is recovered which may slow down actor recovery but only by a small amount if you follow the offline steps.
  
-Migration requires some downtime but the new mechanism removes the restriction of only 3 tags 
-and improves performance, scalability and reliability of eventsByTag queries.
+The `EventsByTagMigration` class provides a set of tools to assist in the migration.
 
-* Migrate data to new table
+The first two are schema changes that should be performed once and can be done while your application is running with the old version of this plugin:
+* `createTables` creates the two new tables required
+* `addTagsColumn` adds a `set<text>` column to the `messages` table 
+ 
+Next is the data migration into a new table that stores events indexed by tag. This will be a slow process as it needs
```

Comment:
can't see that the `EventsByTagMigration` class is mentioned here? perhaps a short code example showing which method to call.

perhaps also mention that it should only be started at one place, so that they don't run it from all cluster nodes

Reply:
I'll add an example

Reply:
I did say "should be performed once and can be done with your application running" i'll add in once on a single node so it is super clear